### DISCUSSION TITLE
How to have multiple webpack in my turborepo apps?

### DISCUSSION THREAD

**Discussion by mahadeer** [vercel/turborepo#6283] [Help] [answered]
I have 3 project inside app folder,

App 1 uses webpack 5, App 2 and 3 uses webpack 4. I have webpack 4 config on root package.json and webpack 5 in App1 package.json.

But the trubo build for App1 is taking webpack 4 always.

How to make App1 to take webpack@5 ??

**Comment by mehulkar** [comment, marked as answer]
> I have webpack 4 config on root package.json

This is likely the issue. I would try removing your webpack configuration from the root package.json and instead localizing your webpack configuration to each app directory. 

This is unrelated to Turborepo, so I will close this down.
